core/shared/src/main/scala/zio/prelude/NonEmptyList.scala

```diff
@@ -620,3 +635,23 @@ trait LowPriorityNonEmptyListImplicits {
   implicit def NonEmptyListPartialOrd[A: PartialOrd]: PartialOrd[NonEmptyList[A]] =
     PartialOrd[List[A]].contramap(_.toList)
 }
+
+trait NonEmptyListSyntax {
+  implicit class NonEmptyListListOps[A](self: List[A]) {
+
+    /**
+     * Converts to a `NonEmptyList` or `None` if empty.
+     */
+    def toNonEmptyList: Option[NonEmptyList[A]] = self match {
```

Comment:
Implementation can just be `fromIterableOption`.

Reply:
Nice! Thank you.